RecoLocalTracker/SiPhase2Clusterizer/test/ClustersValidation.cc

```diff
@@ -33,6 +33,8 @@
 #include "SimDataFormats/Vertex/interface/SimVertexContainer.h"
 #include "SimDataFormats/TrackingHit/interface/PSimHitContainer.h"
 
+#include "SimTracker/SiPhase2Digitizer/interface/Phase2TrackerDigiCommon.h"
```

Comment:
@thomaslenzi 
Yes, the comment was about somewhat improper location of Phase2TrackerDigiCommon.
If you are not the main developer there, please pass this along.
It doesn't have to be addressed with this PR, but better be, eventually.